src/oge/data_pipeline.py

```diff
@@ -142,6 +142,7 @@ def main(args):
     logger.info("1. Downloading data")
     # PUDL
     download_data.download_pudl_data(source="aws")
+    logger.info(f"Using {os.getenv('PUDL_BUILD')} PUDL build")
```

Comment:
I think you want to handle the case where `PUDL_BUILD` is not set:
```python
logger.info(f"Using {os.getenv('PUDL_BUILD', 'stable')} PUDL build")
```

Reply:
Updated